util/syncpacket/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
```

Comment:
Could you add some tests for the crate?

Reply:
I just pushed some tests in the case when syntax is right, then everything should work as expected. I will have a look at `syn` to see how we can test wrong syntax.